bluesky/simulators.py

```diff
@@ -88,28 +84,13 @@ def check_limits(plan):
     ----------
     plan : iterable
         Must yield `Msg` objects
-
-    Raises
-    ------
-    LimitsExceeded
     """
     ignore = []
     for msg in plan:
-        if msg.command == 'set':
-            if msg.obj in ignore:
-                continue  # we have already warned about this device
-            try:
-                low, high = msg.obj.limits
-            except AttributeError:
-                warn("Limits of {} are unknown and can't be checked.".format(msg.obj.name))
-                ignore.append(msg.obj)
-                continue
-            if low == high:
-                warn("Limits are not set on {}".format(msg.obj.name))
+        if msg.command == 'set' and msg.obj not in ignore:
+            if hasattr(msg.obj, "limits"):
```

Comment:
should his be checking for `"check_value"` instead of `"limits"`

Reply:
I agree.  This assumes nothing else.

Reply:
Can we use an f-string in the `warn()` here?

Reply:
The line length is 79.  f-string would save 8 characters.